src/plugins/orangehrmRecruitmentPlugin/Api/CandidateHistoryAPI.php

```diff
@@ -194,6 +194,7 @@ public function getValidationRuleForUpdate(): ParamRuleCollection
                     self::PARAMETER_NOTE,
                     new Rule(Rules::STRING_TYPE)
```

Comment:
don't we have FE text length validation? please add length validation

Reply:
oops

Reply:
done